internal/cli/atlas/streams/connection/connection.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This code was autogenerated at 2023-07-13T05:55:59Z. Note: Manual updates are allowed, but may be overwritten.
+
+package connection
+
+import (
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	const use = "connection"
+	cmd := &cobra.Command{
+		Use:     use,
+		Aliases: cli.GenerateAliases(use),
+		Short:   "Atlas Stream Processor connections.",
+		Long:    `Create, list, update and delete your Atlas Stream Processor connections`,
+	}
+
+	cmd.AddCommand(
+		ListBuilder(),
+		DescribeBuilder(),
```

Comment:
Do we want to make these commands available to customers? If not we can use `Hidden: true` to the command we want to hide

Reply:
I've marked the top level `streams` as hidden, if you know it exists, I guess it's fine to leave the rest as not-hidden?